frontend/src/authentication/ProfilePage.jsx

```diff
@@ -0,0 +1,389 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+import {
+  Box,
+  Text,
+  Stack,
+  HStack,
+  Flex,
+  AbsoluteCenter,
+  Button,
+  useDisclosure,
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  ModalCloseButton,
+  CheckboxGroup,
+  Checkbox,
+  Radio,
+  RadioGroup,
+} from "@chakra-ui/react";
+import { useParams } from "react-router-dom";
+import BackButton from "../app/BackButton";
+import Nav from "../app/Nav";
+import Footer from "../app/Footer";
+
+function ProfilePage() {
+  const [userInfo, setUserInfo] = useState([]);
+  const [isUser, setIsUser] = useState(false);
+  const [newFullName, setNewFullName] = useState("");
+  const [newImageUrl, setNewImageUrl] = useState("");
+  const [categories, setCategories] = useState([]);
```

Comment:
nit: `categories` is pretty vague, I think something like `interests` would better describe the purpose of this variable

Reply:
Thanks for the feedback. Updated in the newest commit: https://github.com/izuchukwumba/peered/pull/8/commits/5fe0755ce9404ede218e0bb4bcfe4fbc765b2e99